### PR TITLE
automatically migrate server list when updating off of beta

### PR DESCRIPTION
**Pull request by trevj**
Thanks for prodding me to do this. Pretty easy in the end, tested on Win 7/8/10.

### CODE REVIEW

File: electron/index.ts [hunk 1]
```diff
@@ -25,6 +28,57 @@ import * as errors from '../www/model/errors';
...
+  try {
+    // The parent folder, 'Local Storage', does not exist at this point.
+    fs.mkdirSync(path.dirname(path.join(app.getPath('userData'), ...suffix)));
+
+    // TODO: Use fs.copyFileSync once we move to an Electron with Node.js 8.5+:
+    //       https://nodejs.org/dist/latest-v8.x/docs/api/fs.html#fs_fs_copyfilesync_src_dest_flags
+    fsextra.copySync(src, dest, {overwrite: true, errorOnExist: true});
+    console.info('migrated beta servers');
+  } catch (e) {
+    // Do *not* the error since it almost certainly contains the PII-type info, viz. username.
```
Comment (alalamav) [inline electron/index.ts]: nit: missing "log"

File: electron/index.ts [hunk 2]
```diff
@@ -25,6 +28,57 @@ import * as errors from '../www/model/errors';
 import {ConnectionStore, SerializableConnection} from './connection_store';
 import * as process_manager from './process_manager';
 
+// Imports the server list from Outline Beta, if one exists, by copying localstorage from Outline
```
Comment (alalamav) [inline electron/index.ts]: Is it worth moving this code to a separate file?

File: electron/index.ts (hunk 1 above)
Comment (trevj) [reply electron/index.ts]: Done.

File: electron/index.ts (hunk 2 above)
Comment (trevj) [reply electron/index.ts]: Mmm, good idea. Moved to `beta_migration.ts`.

**Comment by trevj** [comment]
@alalamav Thanks for the fast review.

**Pull request merged**